pkg/restic/exec_commands.go

```diff
@@ -97,18 +97,20 @@ func RunBackup(backupCmd *Command, log logrus.FieldLogger, updateFunc func(veler
 			select {
 			case <-ticker.C:
 				lastLine := getLastLine(stdoutBuf.Bytes())
-				stat, err := decodeBackupStatusLine(lastLine)
-				if err != nil {
-					log.WithError(err).Errorf("error getting restic backup progress")
-				}
-
-				// if the line contains a non-empty bytes_done field, we can update the
-				// caller with the progress
-				if stat.BytesDone != 0 {
-					updateFunc(velerov1api.PodVolumeOperationProgress{
-						TotalBytes: stat.TotalBytes,
-						BytesDone:  stat.BytesDone,
-					})
+				if len(lastLine) > 0 {
```

Comment:
it'd be more consistent with go style to invert this, so e.g. 

```go
if len(lastLine) == 0 {
    break
}

...
```

so the main logic can be less indented. this is not a merge-blocking comment, though.